types/src/errors/auth.rs

```diff
@@ -30,4 +30,10 @@ pub enum Error {
     /// User does not have permission to a device
     #[error("user does not have permission to a specified device")]
     NoDevicePermission,
+    /// Invalid Google JWT
+    #[error("invalid Google JWT: {0}")]
+    InvalidGoogleJwt(String),
+    /// The CSRF token cookie was missing, or didn't match the token in the request.
+    #[error("Missing or invalid CSRF token")]
+    InvalidCsrfToken,
```

Comment:
maybe we could use existing `InvalidToken(super::token::Error)` variant?

Reply:
I'd rather keep the CSRF token error separate, as it isn't really part of the JWT bit, and it's useful to know the difference when debugging.